kerastuner/tuners/ultraband.py

```diff
@@ -37,3 +229,55 @@ def __init__(self,
             objective,
             max_trials,
             **kwargs)
+
+    def run_trial(self, trial, hp, fit_args, fit_kwargs):
```

Comment:
You almost certainly don't need to copy this method. It would be bad to override it because 90% of this code is redundant with the code on the base class.

IMO you should be able to:
1) override `run_trial`
2) patch `epochs` in fit_kwargs
3) call super `run_trial`